application/common/sync/Synchronizer.php

```diff
@@ -411,7 +411,12 @@ public function syncAll()
             $this->idStore->getIdStoreName()
         ));
 
-        $idStoreUsers = $this->idStore->getAllActiveUsers();
+        try {
+            $idStoreUsers = $this->idStore->getAllActiveUsers();
+        } catch (Exception $e) {
+            throw new Exception('Failed to get active users from ID Store: ' . $e->getMessage(), 1669770777);
```

Comment:
The code (1669770777) is "arbitrary". I'd do it without the code. Also, an sprintf() is cleaner than a concatenation.

Reply:
I've never completely understood why the IDP code uses arbitrary exception codes. I guess it's to decrease the chances of two exceptions that look exactly the same. But I'm going to leave it since that's the standard for this repo.